cli/src/commands/version.rs

```diff
@@ -22,14 +22,71 @@ use crate::ui::Ui;
 
 /// Display version information
 #[derive(clap::Args, Clone, Debug)]
-pub(crate) struct VersionArgs {}
+pub(crate) struct VersionArgs {
+    /// Display only the version number and nothing else.
+    #[arg(long)]
+    pub(crate) numeric_only: bool,
+
+    /// Display build information.
+    #[arg(long, conflicts_with = "numeric_only")]
+    pub(crate) details: bool,
+}
```

Comment:
Alternatively, we can add `-Ttemplate`. I think typical use case would be `-Tversion` or something to get only the numeric part.

Reply:
I didn't think about templates. But it's probably not worth it IMO. In all the years on GHC, I think we only ever needed the `--numeric-version` for things like scripts, and `--details` for things like bug reports.